skimage/transform/_warps.py

```diff
@@ -756,7 +756,7 @@ def _clip_warp_output(input_image, output_image, mode, cval, clip):
             min_val = min(min_val, cval)
             max_val = max(max_val, cval)
 
-        np.clip(output_image, min_val, max_val, out=output_image)
+        np.clip(output_image, np.array(min_val), np.array(max_val), out=output_image)
```

Comment:
```suggestion
        # Force other array types to NumPy array (gh-7159)
        min_val = np.asarray(min_val)
        max_val = np.asarray(max_val)
        np.clip(output_image, min_val, max_val, out=output_image)
```
In lack of a test preserving this intentional change, maybe we want to add a comment? Not sure if `asarray` is needed here but it may prevent unecessary copies?